src/FunctionApps/python/IntakePipeline/README.md

```diff
@@ -2,13 +2,27 @@
 
 ## Config
 
-This function app requires some config to be present. In production this is defined on the function app, and locally, you'll want to add
+### IngestPipeline
+This function app module requires some config to be present. In production this is defined on the function app, and locally, you'll want to add
 items in `local.settings.json` under `Values`.
 
-* `INTAKE_CONTAINER_URL`: a container URL containing ndjson formatted FHIR records (eg: https://pitestdatasa.blob.core.windows.net/bronze)
-* `INTAKE_CONTAINER_PREFIX`: a string prefix so we're not scanning the entire container (eg: decrypted/valid-messages)
-* `OUTPUT_CONTAINER_PATH`: the blob container path to store processed items
+* `INTAKE_CONTAINER_URL`: a container URL containing FHIR bundles (eg: https://pitestdatasa.blob.core.windows.net/bronze)\
+*Note:* Stored bundles must be of type `batch`.  Entries in the batch Bundle should contain a valid `request` describing how the `resource` should be submitted to the FHIR Server.
```

Comment:
I'd almost make these notes a separate subheader dealing with paths/contents and whatnot.

Reply:
Updated, and separated content into IntakePipeline and GenerateCSV specific READMEs.